### DISCUSSION TITLE
Setup on Pi Zero W.  RA needs Node v12, but Pi Zero W only supports upto v11

### DISCUSSION THREAD

**Discussion by johntamplin** [mKeRix/room-assistant#1157] [General]
Trying to setup on a Raspberry Pi Zero, but there is a conflict: Pi Zero W use ARMV6 for which 'node' version is only available up to v11.15.0. However, room-assistant requires node v12 or higher.  I tried with v11.15.0 but it doesn't work.  Installed fine on a Pi v4 (ARMV7) so I think I'm doing the install correctly.   Has anyone successfully installed room-assistant on Pi Zero W and if so how did you get around this conflict?

**Comment by paulrbeers81** [comment]
Did you follow the specific Pi Zero W instructions that is linked from the main install page?   I just ran these on Monday with no issues.....

https://www.room-assistant.io/guide/quickstart-pi-zero-w.html#installing-room-assistant

That is there specifically because of this limitation.....
